test/jinahub/encoder/image/test_clip.py

```diff
@@ -0,0 +1,11 @@
+from jina import Flow, Document
+
+from jinahub.encoder.image.clip import CLIPImageEncoder
+import numpy as np
+
+def test_clip():
+    def on_done(resp):
+        print('assert', resp)
```

Comment:
this test did not actually test the encoder, but the flow

Reply:
:) I just committed a template without content as a draft. It was not intended to work.